molecule/command/lint.py

```diff
@@ -23,13 +23,19 @@
 
 from molecule import logger
 from molecule.command import base
+from molecule import util
+
 
 LOG = logger.get_logger(__name__)
 
 
 class Lint(base.Base):
     """
-    Lint Command Class.
+    Lint command executes external linters.
+
+    You need to remember to install those linters. For convenience, there is a
+    package extra that installs the most common ones, use it like
+    ``pip install "molecule[extra]"``.
```

Comment:
Should this be ``pip install molecule[lint]`` rather than ``pip install molecule[extra]`` ?

Reply:
Indeed.